hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/acl/OzoneNativeAuthorizer.java

```diff
@@ -101,11 +100,18 @@ public boolean checkAccess(IOzoneObj ozObject, RequestContext context)
     boolean isListAllVolume = ((context.getAclRights() == ACLType.LIST) &&
         objInfo.getVolumeName().equals(OzoneConsts.OZONE_ROOT));
     if (isListAllVolume) {
-      return getAllowListAllVolumes();
+      return getAllowListAllVolumes()
+          || isAdmin(ozSuperReadAdmins, context.getClientUgi());
     }
 
     ACLType parentAclRight = OzoneAclUtils.getParentNativeAcl(
         context.getAclRights(), objInfo.getResourceType());
+
+    // bypass only read checks for super read
+    if (isAdmin(ozSuperReadAdmins, context.getClientUgi())
+        && parentAclRight == ACLType.READ) {
+      return true;
```

Comment:
Both parentAclRight and object itself acl should be checked. We cannot return here.

Reply:
@z-bb Please add a unit test in TestOzoneAdministrators, testing operation on bucket, which will cover both the parent and child acl verification logic.

Reply:
The above code is deprecat.
current latest code is
```
boolean isSuperReadAdmin = isAdmin(ozSuperReadAdmins,
        context.getClientUgi());

    // bypass read checks for super read users
    if (isSuperReadAdmin && (context.getAclRights() == ACLType.READ
        || context.getAclRights() == ACLType.READ_ACL
        || context.getAclRights() == ACLType.LIST)) {
      return true;
    }
```
Do you think this is a problem @ChenSammi

Reply:
later, I will add testing operation on bucket